pkg/controller/nodeipam/ipam/cloud_cidr_allocator.go

```diff
@@ -306,6 +309,27 @@ func (ca *cloudCIDRAllocator) updateCIDRAllocation(nodeName string) error {
 	return err
 }
 
+func updateNodePodCIDRs(node *v1.Node, podCIDRs []string) bool {
```

Comment:
Can we call this `needUpdateCIDRs` or something 

verbMethods usually have side effects.

Reply:
Done. Named it as needPodCIDRsUpdate